src/server/game/Spells/Spell.cpp

```diff
@@ -1586,6 +1586,15 @@ void Spell::SelectImplicitCasterDestTargets(SpellEffIndex effIndex, SpellImplici
                                 destz = prevZ;
                             //LOG_ERROR("spells", "(collision) destZ rewrited in prevZ");
 
+                            // Don't make the player move backward from the xy adjustments by collisions.
+                            if ((DELTA_X > 0 && startx > destx) || (DELTA_X < 0 && startx < destx)
+                            ||  (DELTA_Y > 0 && starty > desty) || (DELTA_Y < 0 && starty < desty))
```

Comment:
```suggestion
                            if ((DELTA_X > 0 && startx > destx) || (DELTA_X < 0 && startx < destx)
                                ||  (DELTA_Y > 0 && starty > desty) || (DELTA_Y < 0 && starty < desty))
```

Reply:
I made the spacing this way so they would be perfectly alligned for readability purposes.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/22718174/7fb2fae0-2f45-4ec6-a10b-1a4b83702680)

With the change you are offering, there would be a double space right here.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/22718174/1fd555d5-a4ca-4d06-bfc9-ea2337cf0f7e)

I do prefer not changing it but if we must, I'd remove the double space or making it a single hard-to-read line.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/22718174/0c644490-e3e3-402c-8970-8e06e4c4a71d)

What do you prefer? Making it a single line or removing the double space?

Reply:
Remove the double space, I had missed that initially

Reply:
What you could do is moeve the || to the previous line for readability

Reply:
I did not think of that, I love it. :D